README.md

```diff
@@ -1,2 +1,33 @@
-# AppliedAlgorithms
-Guides and solutions to various types of programming problems.
+# TTU Applied Algorithms
+Name under construction.
+
+# Overview
+This repo serves as a central location for all things related to solving competitive programming problems. Here is where you can find guides and solutions to various types of programming problems in addition to tutorials on different types of problems.
+
+# Purpose
+The purpose of this group is to bring experienced and beginner programmers together in a relaxed environment. Here the experienced programmers can store not just their solutions to various problems found online, but a detailed solution guide that is intended to walk beginners through their thought process and to show them how they solved the problem. We hope that by doing this, beginner programmers can get past their fear of these problems and start to enjoy them.
+
+We, the creators of this group, understand that at first, these problems can seem very intimidating and at first, one can feel like they lack the skills needed to solve these problems. Our hope, with this project, is to eliminate those fears and ideas and instead, show how it's not too difficult to get involved in and it doesn't require too much upfront knowledge to get started.
+
+Again, this repo is designed for beginners so we will have guides and resources at every step of the way. No matter you skill level, you _can_ get into online programming problems.
+
+# F.A.Q.
+We understand that you might have a lot of questions concerning this and we hope to answer them all. If you have a question that is not answered below, feel free to contact us and we will try our best to answer you!
+
+## Why do I want to do these online problems?
+Are you a Computer Science or related major? Do you want a good software engineering job? Do you want to keep your programming skills up to date? These types of problems can help with all of that and more!
+Here is what you can get out working on these types of problems:
+- **Technical interview prep**: These types of problems are _very_ similar to how technical interview questions are structured. So if you plan on interviewing for a technical internship or job, these questions are the best way to prepare yourself.
+
+- **Keeping your programming skills sharp and refined**: Since these problems are short and require a solid knowledge of algorithms and data structures, by working on these problems regularly, you can keep not just your programming knowledge of languages you know, but use these problems to gain skills in languages you aren't familiar with.
+
+- **It's fun**: This one might be harder to believe, but after you keep really into these types of problems they can seem fun. If you end up really enjoying these problems, you might want to get involved in online programming competitions! And hey, that's something else you can put on your resume.
+
```

Comment:
What do you think is more important to stress? The fun (which I do agree with, yet seem to be in a minority) or the importance it can have on technical interviews?